cpp/libcudf_kafka/tests/CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ function(ConfigureTest test_name)
     ${test_name}
     PROPERTIES RUNTIME_OUTPUT_DIRECTORY "$<BUILD_INTERFACE:${CUDF_KAFKA_BINARY_DIR}/gtests>"
                INSTALL_RPATH "\$ORIGIN/../../../lib"
+               CXX_STANDARD 20
```

Comment:
Should we adopt the same pattern of setting C++ standard globally here like above as well?

Reply:
No idea what the difference is, very much open to suggestions.